### DISCUSSION TITLE
Hackathon: 2024-10-29

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#849] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by JessicaS11** [comment]
See [this post ](https://github.com/nsidc/earthaccess/discussions/847#discussioncomment-11091500)(in its parent discussion thread) summarizing today's work on returning a results class instead of a list.

We also got team members set up with a working dev environment (non-trivial on Windows!) and started adding some properties to the `DataCollections` class.

**Comment by danielfromearth** [comment]
@asteiker and I went through and tidied up old issues—including adding comments, adding labels, closing the issue, and converting to a discussion.  I think we made it through a couple dozen issues, and left off on what is currently Page 4 of the issues.

**Reply by asteiker** [reply]
Yes! And we also started to identify a few themes and clusters of related issues:

- EULA behavior (not necessarily a lot of issues on this, but more of a long-standing issue that remains to be fully resolved)
- S3 access (e.g.  #231 )
- OPeNDAP URLs and service usage
